README.md

```diff
@@ -2,6 +2,35 @@
 
 A web-based diagram client framework for the [Graphical Language Server Platform (GLSP)](https://github.com/eclipse-glsp/glsp) based on [Eclipse Sprotty](https://github.com/eclipse/sprotty).
 
+## Workflow Diagram Example
+
+The workflow diagram is a consistent example provided by all GLSP components. The example implements a simple flow chart diagram editor with different types of nodes and edges (see screenshot below). The example can be used to try out different GLSP features, as well as several available integrations with IDE platforms (Theia, VSCode, Eclipse, Standalone).
+As the example is fully open source, you can also use it as a blueprint for a custom implementation of a GLSP diagram editor.
+See [our project website](https://www.eclipse.org/glsp/documentation/#workflowoverview) for an overview of the workflow example and all components implementing it.
+
+![Workflow Diagram](/documentation/standalone-diagram.gif)
+
+### How to start the Workflow Diagram example?
+
+First, you need to build the Client packages:
+
+```
+yarn install
+```
+
+And the GLSP Server (Which includes the Workflow Diagram Server example). See [`glsp-server`](https://github.com/eclipse-glsp/glsp-server#building) for instructions.
+
+Once both the Server and the Client packages are built, you can start both:
+
+- Server: in `examples/org.eclipse.glsp.example.workflow`, start `org.eclipse.glsp.example.workflow.launch.WorkflowServerLauncher.java` with these arguments: `--websocket --port=8081` (Or use the included `Start_Workflow_Example_Server_(Websocket).launch` Launch Configuration from Eclipse).
```

Comment:
Maybe it is worth mentioning that the server can also be simply started from command line after building as documented here: https://github.com/eclipse-glsp/glsp#building-and-running-the-glsp-backend
This way, interested users wouldn't have to set up an IDE?

Reply:
There are too many readmes spread all around :) I wasn't aware of this part of the doc